packages/thailand/thailand.svg

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created for MapSVG plugin: http://mapsvg.com -->
+<svg
+   xmlns:mapsvg="http://mapsvg.com"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   mapsvg:geoViewBox="97.344728 20.463430 105.640023 5.614417"
+   width="559.57092"
+   height="1024.7631">
```

Comment:
Could you remove the unnecessary attributes and set the `viewBox` please? You can look at other SVG maps